tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -179,5 +179,27 @@ The expected failures in this file are from features in the owncloud/ocis repo.
+- [apiSpaces/setQuota.feature:80](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L80)
+- [apiSpaces/setQuota.feature:81](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L81)
+- [apiSpaces/setQuota.feature:82](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L82)
+- [apiSpaces/setQuota.feature:83](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L83)
+- [apiSpaces/setQuota.feature:84](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L84)
+- [apiSpaces/setQuota.feature:85](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L85)
+- [apiSpaces/setQuota.feature:86](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L86)
+- [apiSpaces/setQuota.feature:87](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L87)
+- [apiSpaces/setQuota.feature:88](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L88)
+- [apiSpaces/setQuota.feature:89](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L89)
+- [apiSpaces/setQuota.feature:90](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L90)
+- [apiSpaces/setQuota.feature:91](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L91)
+- [apiSpaces/setQuota.feature:173](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L173)
+- [apiSpaces/setQuota.feature:174](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L174)
+- [apiSpaces/setQuota.feature:175](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L175)
+- [apiSpaces/setQuota.feature:176](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L176)
+- [apiSpaces/setQuota.feature:177](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L177)
+- [apiSpaces/setQuota.feature:178](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L178)
+- [apiSpaces/setQuota.feature:239](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L239)
+- [apiSpaces/setQuota.feature:240](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L240)
+- [apiSpaces/setQuota.feature:241](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/setQuota.feature#L241)
```

Comment:
Duplication of ### [API requests from an unauthorized user should return 403](https://github.com/owncloud/ocis/issues/5938) in line 136